src/pages/use-cases/service-mesh.jsx

```diff
@@ -51,13 +54,47 @@ const sectionContent4 = {
   imageRight: false,
 };
 
+const testimonials = [
+  {
+    logo: RocheLogo,
+    title: 'How Roche Manages Network Connectivity for 1000+ Edge Clusters',
+    CTAs: [
+      {
+        CTAtext: 'Watch The Talk',
+        url: 'https://www.youtube.com/watch?v=vKgRf4OzTIE',
+      },
+    ],
+    description:
+      "The Roche team utilized Cilium's service mesh as a solution for managing network connectivity across their edge clusters. Cilium enabled them to have more fine-grained traffic control, and simplified the operations and configuration of network policies using a GitOps approach, alongside the deployment of workloads",
+  },
+  {
+    logo: SicrediLogo,
+    title: 'Strengthening Security Across Distributed Kubernetes Clusters',
+    CTAs: [
+      {
+        CTAtext: 'Watch The Talk',
+        url: 'https://youtu.be/MSqI-gBiCrc?si=ktIHXkxhuHpuwC6a',
+      },
+      // {
+      //   CTAtext: 'Read Case Study',
+      //   url: '',
+      // },
+    ],
+    description:
+      "We started using Cilium in our EKS clusters and encountered security challenges. So many microservices were running without any kind of authentication or encryption. This prompted us to start doing a service mesh proof of concept. Although we already had Cilium available, we also evaluated Consul Connect, Kong Mesh, and Istio. Cilium stood out as it did not require sidecars, appealing to us not only for resource savings but also to avoid potential load issues caused by additional software. Cilium's sidecar-less approach did call our attention, leading us to adopt it to provide security across all our microservices.",
+    quote: true,
+    person: 'Matheus Morais - IT Infrastructure Analyst, Sicredi',
+  },
+];
+
```

Comment:
you should also add the NYT talk to the service mesh page

Reply:
Alright, would do!